app/app/config.py

```diff
@@ -0,0 +1,9 @@
+from vyper import v
+
```

Comment:
I think you should add the configuration and configuration aggregate class because they facilitate communicating your expected configuration interface.

Reply:
done